draft-ietf-ppm-dap.md

```diff
@@ -852,14 +867,21 @@ provide for some small leeway, usually no more than a few minutes, to account
 for clock skew. If the leader rejects a report for this reason, it SHOULD abort
      the upload protocol and alert the client with error "reportTooEarly".
 
+If the Report contains an unrecognized exgension, then the Leader MAY abort the
+upload request with error "unrecognizedExtension". Note that this behavior is
+not mandatory because it requires the Leader to decrypt its input share.
```

Comment:
This is a bit awkward. The requirement I'm trying to meet here is: The Leader shouldn't have to have its HPKE decryption keys available during the upload sub-protocol. One way to avoid this would be to move the extensions out of the plaintext.